src/olympia/devhub/templates/devhub/includes/macros.html

```diff
@@ -10,6 +10,16 @@
 </p>
 {% endmacro %}
 
+{% macro markdown_tip(title=None) %}
+<p class="syntax-support">
+{# L10n: %s is a list of markdown syntax. #}
+<span class="tooltip" title="{% if title %}{{ title }}{% else %}{{
+      _('Allowed Markdown: %(allowed_markdown)s', allowed_markdown='links, abbr,  bold, italics, blockquote, code, ol, ul')
```

Comment:
allowed_markdown should be in a constant somewhere. If these are in fact the only tags allowed then use that same constant in actual processing.

Reply:
Just since you pointed this out --

> There should probably be a more informative way to explain what syntax or Markdown is supported, possibly similar to how GitHub explains it via the 'Markdown is Supported' button by navigating to a [dedicated page](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

I mentioned this to Andrew offhandedly as, especially with the extensions, it may not be obvious what Markdown syntax is allowed in what way, and he mentioned maybe making it a page on the extension workshop over the tooltip. What do you think?

Reply:
I'm definitely in favor of that yes. I don't think it is directly related to my ask though. I want to make sure that the tags specified in this tooltip are "bound" to the tags we actually filter for, so that if one were to change, the other is automatically synced.

It would be a bad UX if the tooltip allowed links, but actually we don't. Would it be possible to extract this list to a constant and the `", ".join(ALLOWED_MARKDOWN_TAGS)` in the template?

And then where we actually validate the input, we can use the same constant.

Reply:
It looks like we have such a constant in the translations model file already "allowed_tags"

Reply:
I had meant it in a replace-the-tooltip-entirely way, but I figure there's no harm in having both.